tests/env_vars_test.rs

```diff
@@ -20,11 +20,23 @@ mod env_var_mask {
 $MASK test
 ~~~
 
+~~~powershell
+$path = $env:MASK.replace("\\?\", "")
+$pos = $path.IndexOf(" ");
+$arglist = $path.Substring($pos + 1);
+
+Start-Process mask.exe -ArgumentList "$arglist test" -wait -NoNewWindow -PassThru
+~~~
```

Comment:
> The Paths in windows are proceeded by \\?\, for instance: \\?\C:\\path\\to\\maskfile.md. If this prefix is removed from the path, it will work without issue in Mask/Rust. Only had to do this one time and it was done via Powershell.

I think your comment refers to this right?

So this is just something people will need to do when they try to call `mask` again? I think this is fine for now. If Windows devs start complaining about how painful this is, we can maybe normalize `$MASK` for Windows so that it works in a way that is more idiomatic for powershell. Don't need to do this today though.

Reply:
Yeah this was my thought too.  I decided that it would be easier for now to just modify the path via the script rather then to dig into the rust code.